### PR TITLE
New default preferences

### PR DESCRIPTION
**Pull request by brandelune**
This PR keep the preferences related commits from https://github.com/omegat-org/omegat/pull/386

### CODE REVIEW

File: src/org/omegat/gui/preferences/view/GeneralOptionsController.java [hunk 1]
```diff
@@ -84,13 +84,13 @@ private void openFile(File path) {
     @Override
     protected void initFromPrefs() {
         panel.tabAdvanceCheckBox.setSelected(Core.getEditor().getSettings().isUseTabForAdvance());
-        panel.confirmQuitCheckBox.setSelected(Preferences.isPreference(Preferences.ALWAYS_CONFIRM_QUIT));
+        panel.confirmQuitCheckBox.setSelected(!Preferences.isPreference(Preferences.ALWAYS_CONFIRM_QUIT));
```
Comment (miurahr) [inline src/org/omegat/gui/preferences/view/GeneralOptionsController.java:87]: It should not be reverted, that become a logic bug. 

```suggestion
        panel.confirmQuitCheckBox.setSelected(Preferences.isPreferenceDefault(Preferences.ALWAYS_CONFIRM_QUIT, true));
```

File: src/org/omegat/gui/preferences/view/HistoryAutoCompleterOptionsController.java [hunk 2]
```diff
@@ -59,15 +59,15 @@ private void initGui() {
     @Override
     protected void initFromPrefs() {
         panel.historyCompletionCheckBox
-                .setSelected(Preferences.isPreference(Preferences.AC_HISTORY_COMPLETION_ENABLED));
+                .setSelected(!Preferences.isPreference(Preferences.AC_HISTORY_COMPLETION_ENABLED));
```
Comment (miurahr) [inline src/org/omegat/gui/preferences/view/HistoryAutoCompleterOptionsController.java:62]:
```suggestion
                .setSelected(Preferences.isPreferenceDefualt(Preferences.AC_HISTORY_COMPLETION_ENABLED, true));
```

File: src/org/omegat/gui/preferences/view/HistoryAutoCompleterOptionsController.java [hunk 3]
```diff
@@ -59,15 +59,15 @@ private void initGui() {
...
         panel.historyPredictionCheckBox
-                .setSelected(Preferences.isPreference(Preferences.AC_HISTORY_PREDICTION_ENABLED));
+                .setSelected(!Preferences.isPreference(Preferences.AC_HISTORY_PREDICTION_ENABLED));
```
Comment (miurahr) [inline src/org/omegat/gui/preferences/view/HistoryAutoCompleterOptionsController.java:64]:
```suggestion
                .setSelected(Preferences.isPreferenceDefault(Preferences.AC_HISTORY_PREDICTION_ENABLED, true));
```

File: src/org/omegat/gui/preferences/view/SpellcheckerConfigurationController.java [hunk 4]
```diff
@@ -276,7 +276,7 @@ protected Language getCurrentLanguage() {
 
     @Override
     protected void initFromPrefs() {
-        panel.autoSpellcheckCheckBox.setSelected(Preferences.isPreference(Preferences.ALLOW_AUTO_SPELLCHECKING));
+        panel.autoSpellcheckCheckBox.setSelected(!Preferences.isPreference(Preferences.ALLOW_AUTO_SPELLCHECKING));
```
Comment (miurahr) [inline src/org/omegat/gui/preferences/view/SpellcheckerConfigurationController.java:279]:
```suggestion
        panel.autoSpellcheckCheckBox.setSelected(Preferences.isPreferenceDefault(Preferences.ALLOW_AUTO_SPELLCHECKING, true));
```

File: src/org/omegat/gui/preferences/view/EditingBehaviorController.java [hunk 5]
```diff
@@ -84,11 +84,11 @@ protected void initFromPrefs() {
             panel.prefixText.setText(Preferences.getPreference(Preferences.BEST_MATCH_EXPLANATORY_TEXT));
         }
 
-        panel.allowTranslationEqualToSource.setSelected(Preferences.isPreference(Preferences.ALLOW_TRANS_EQUAL_TO_SRC));
+        panel.allowTranslationEqualToSource.setSelected(!Preferences.isPreference(Preferences.ALLOW_TRANS_EQUAL_TO_SRC));
         panel.exportCurrentSegment.setSelected(Preferences.isPreference(Preferences.EXPORT_CURRENT_SEGMENT));
         panel.stopOnAlternativeTranslation
                 .setSelected(Preferences.isPreference(Preferences.STOP_ON_ALTERNATIVE_TRANSLATION));
-        panel.convertNumbers.setSelected(Preferences.isPreference(Preferences.CONVERT_NUMBERS));
+        panel.convertNumbers.setSelected(!Preferences.isPreference(Preferences.CONVERT_NUMBERS));
```
Comment (miurahr) [inline src/org/omegat/gui/preferences/view/EditingBehaviorController.java:91]:
```suggestion
        panel.convertNumbers.setSelected(Preferences.isPreferenceDefault(Preferences.CONVERT_NUMBERS, true));
```

File: src/org/omegat/gui/preferences/view/EditingBehaviorController.java [hunk 6]
```diff
@@ -84,11 +84,11 @@ protected void initFromPrefs() {
             panel.prefixText.setText(Preferences.getPreference(Preferences.BEST_MATCH_EXPLANATORY_TEXT));
         }
 
-        panel.allowTranslationEqualToSource.setSelected(Preferences.isPreference(Preferences.ALLOW_TRANS_EQUAL_TO_SRC));
+        panel.allowTranslationEqualToSource.setSelected(!Preferences.isPreference(Preferences.ALLOW_TRANS_EQUAL_TO_SRC));
```
Comment (miurahr) [inline src/org/omegat/gui/preferences/view/EditingBehaviorController.java:87]:
```suggestion
        panel.allowTranslationEqualToSource.setSelected(Preferences.isPreferenceDefault(Preferences.ALLOW_TRANS_EQUAL_TO_SRC, true));
```

File: src/org/omegat/gui/main/MainWindowMenu.java [hunk 7]
```diff
@@ -572,11 +572,11 @@ private void updateCheckboxesOnStart() {
         optionsAutoCompleteShowAutomaticallyItem.setSelected(Preferences.isPreferenceDefault(
                 Preferences.AC_SHOW_SUGGESTIONS_AUTOMATICALLY, true));
         optionsAutoCompleteHistoryCompletionMenuItem
-                .setSelected(Preferences.isPreference(Preferences.AC_HISTORY_COMPLETION_ENABLED));
+                .setSelected(!Preferences.isPreference(Preferences.AC_HISTORY_COMPLETION_ENABLED));
         optionsAutoCompleteHistoryPredictionMenuItem
-                .setSelected(Preferences.isPreference(Preferences.AC_HISTORY_PREDICTION_ENABLED));
+                .setSelected(!Preferences.isPreference(Preferences.AC_HISTORY_PREDICTION_ENABLED));
```
Comment (miurahr) [inline src/org/omegat/gui/main/MainWindowMenu.java:577]:
```suggestion
                .setSelected(Preferences.isPreferenceDefault(Preferences.AC_HISTORY_PREDICTION_ENABLED, true));
```

File: src/org/omegat/gui/main/MainWindowMenu.java [hunk 8]
```diff
@@ -572,11 +572,11 @@ private void updateCheckboxesOnStart() {
         optionsAutoCompleteShowAutomaticallyItem.setSelected(Preferences.isPreferenceDefault(
                 Preferences.AC_SHOW_SUGGESTIONS_AUTOMATICALLY, true));
         optionsAutoCompleteHistoryCompletionMenuItem
-                .setSelected(Preferences.isPreference(Preferences.AC_HISTORY_COMPLETION_ENABLED));
+                .setSelected(!Preferences.isPreference(Preferences.AC_HISTORY_COMPLETION_ENABLED));
```
Comment (miurahr) [inline src/org/omegat/gui/main/MainWindowMenu.java:575]:
```suggestion
                .setSelected(Preferences.isPreferenceDefault(Preferences.AC_HISTORY_COMPLETION_ENABLED, true));
```

**Review by miurahr — CHANGES_REQUESTED** [review]
Please consider again how to change default values.

**Comment by brandelune** [comment]
Thank you for your help.

**Comment by brandelune** [comment]
```
> Task :compileJava
/Users/suzume/Documents/Repositories/omegat.brandelune/src/org/omegat/gui/preferences/view/HistoryAutoCompleterOptionsController.java:62: error: cannot find symbol
                .setSelected(Preferences.isPreferenceDefualt(Preferences.AC_HISTORY_COMPLETION_ENABLED, true));
                                        ^
  symbol:   method isPreferenceDefualt(String,boolean)
  location: class Preferences
1 error
```

File: src/org/omegat/gui/preferences/view/HistoryAutoCompleterOptionsController.java (hunk 2 above)
Comment (brandelune) [reply src/org/omegat/gui/preferences/view/HistoryAutoCompleterOptionsController.java]: `isPreferenceDefualt` should be `isPreferenceDefault`

**Review by miurahr — APPROVED** [review]

**Comment by miurahr** [comment]
Now recreate commit history and force push.

**Pull request closed without merging**